ateam_kenobi/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ find_package(control_toolbox REQUIRED)
 find_package(ompl REQUIRED)
 find_package(angles REQUIRED)
 find_package(OpenCV REQUIRED)
+find_package(nlohmann_json REQUIRED)
```

Comment:
huh... how did the play info stuff ever work if we weren't building with the nlohmann package...

Reply:
nlohmann-json is header only, so technically this line isn't strictly required as long as you luck out and some other dependency adds the needed root include path.